docs/design/host_to_services_OpenFlow.md

```diff
@@ -186,11 +221,16 @@ tcp      6 116 TIME_WAIT src=172.18.0.3 dst=172.18.0.4 sport=55978 dport=80 src=
 
 #### Reply
 
-When the reply packet comes back into ovn-worker via the eth0 interface, the packet will be forwarded back into OVN GR (due to a CT match in zone 64000). OVN GR will then handle unSNAT, unDNAT and send the packet back towards breth0 where the packet will be handled the same way as it was in the previous example.
+When the reply packet comes back into ovn-worker via the eth0 interface, the packet will be forwarded back into OVN GR 
+(due to a CT match in zone 64000). OVN GR will then handle unSNAT, unDNAT and send the packet back towards breth0 where
```

Comment:
will greatly help readers to reference both ovs flows and conntrack rule involved when u reference specific packet flow path similar to what u did with request